pkg/features/features.go

```diff
@@ -135,7 +138,7 @@ var allFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	ManagedIstio:       {Default: true, PreRelease: featuregate.Beta},
 	APIServerSNI:       {Default: true, PreRelease: featuregate.Beta},
 	SeedChange:         {Default: true, PreRelease: featuregate.Beta},
-	SeedKubeScheduler:  {Default: false, PreRelease: featuregate.Alpha},
+	SeedKubeScheduler:  {Default: false, PreRelease: featuregate.Deprecated},
```

Comment:
I guess this feature gate can be removed from example yamls as it is deprecated and not recommended to be used.

Reply:
Yes, makes sense!